index.html

```diff
@@ -118,6 +148,78 @@ <h2 class="heading heading_level-2">Популярные товары</h2>
+                                <img src="images/feature-1.svg" alt="">
+                            </div>
+                            <div class="feature-card__description">
+                                Круглосуточная служба поддержки
+                            </div>
+                        </div>
+                    </div>
+                    <div class="column column_s-2 column_m-2 column_l-4">
+                        <div class="feature-card">
+                            <div class="feature-card__image">
+                                <img src="images/feature-2.svg" alt="">
+                            </div>
+                            <div class="feature-card__description">
+                                100% гарантия возврата и подлинность товара
+                            </div>
+                        </div>
+                    </div>
+                    <div class="column column_s-2 column_m-2 column_l-4">
+                        <div class="feature-card">
+                            <div class="feature-card__image">
+                                <img src="images/feature-3.svg" alt="">
+                            </div>
+                            <div class="feature-card__description">
+                                Быстрая доставка по всей России
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <footer class="footer">
+            <div class="column-wrapper">
+                <div class="column-row">
+                    <div class="column column_s-2 column_m-3 column_l-4">
+                        <div class="footer__icons">
+                            <a href="/" class="footer__icon footer__icon_facebook">facebook</a>
+                            <a href="/" class="footer__icon footer__icon_twitter">twitter</a>
+                            <a href="/" class="footer__icon footer__icon_vk">vk</a>
+                            <a href="/" class="footer__icon footer__icon_instagram">instagram</a>                          
+                        </div>
+                    </div>
+                    <div class="column column_s-2 column_m-3 column_l-4">
+                        <div class="footer__buttons">
+                            <button class="footer__button footer__button_app-store">App Store</button>
+                            <button class="footer__button footer__button_google-play">Google Play</button>                         
+                        </div>
+                    </div>
+                      <div class="column column_s-2 column_m-6 column_l-4 column_hidden">
+                          <div class="footer__logo"></div>
+                    </div>
+                    <div class="column column_s-2 column_m-6 column_l-12">
+                        <div class="footer__line"></div>
+                    </div>
```

Comment:
Элементы, которые всегда растягиваются на всю ширину и не требуют колонок, можно и не помещать в них. 
```suggestion
                </div>
                <div class="footer__line"></div>
                <div class="column-row">
```

Reply:
Fixed.